app/mailers/jwt_api_entreprise_mailer.rb

```diff
@@ -3,8 +3,21 @@ def expiration_notice(jwt, nb_days)
     @jwt = jwt
     @nb_days = nb_days
 
+    recipients = exp_notice_recipients
     subject = "API Entreprise - Votre jeton expire dans #{@nb_days} jours !"
 
-    mail(to: jwt.user.email, subject: subject)
+    mail(to: recipients, subject: subject)
+  end
+
+  private
+
+  def exp_notice_recipients
+    # TODO This could be better and clearer here, be patient it will be refactor soon
```

Comment:
Je vais nitpick ici mais je préfère un commentaire qui clutter moins

#TODO refactor is planned

Enfin d'un autre coté si le refactor est planifié, le commentaire va virer...

Reply:
> Je vais nitpick ici mais je préfère un commentaire qui clutter moins

En français ça donne quoi ?

Reply:
Je vais faire mon relou mais je préfère un commentaire qui prend moins de place

Non je n'ai pas demandé de changement, c'est juste pour l'avenir